### ISSUE TITLE
Some questions about test_seg.npz data

### ISSUE THREAD

**Issue by Flowers-M** [voxelmorph/voxelmorph#32]
Dear author, I'm reading your paper published in CVPR. There are some questions I am very confused. Can you give me some help?
1. I saw that you used the test_seg.npz data when testing. I am confused about what it is used for.
2. What kind of processing or software did you used to get this  data?  FreeSurfer or ANTs?
3. If I don't have the data for these test_seg, can I complete the test?
Looking forward to your answer. Thank you.

**Comment by adalca** [comment]
Hi,
1. test_seg.npz is simply a segmentation map for the test volume. With a trained model, you could register the test volume to the atlas, and use the test_seg to check how well the registration went.
2. We used FreeSurfer to preprocess our data and extract labels for evaluation.
3. If you don't have your own training data, you could test using our pre-trained model.

**Comment by Flowers-M** [comment]
Thank you very much for your answer. Is the test_seg.npz data also obtained using FreeSurfer?

**Comment by adalca** [comment]
Yes, it's an example of our processing!

**Comment by hassaanmahmood** [comment]
@adalca hi,
From lablel folder, after running complete recon-all, How can we create test_seg.npz . 
Thanks

**Comment by hassaanmahmood** [comment]
currently I used your label.mat file. 
and from freesurfer
aseg.mgz as test_seg and brain.mgz as moving .... 
thanks

**Comment by adalca** [comment]
@hassaanmahmood,
I'm not 100% sure what you are asking, but essentially we used the `aseg.mgz` files for our testing. `aseg.mgz` has a bunch of segmentations, though, and we sub-selected those segmentations (labels) in `labels.mat` to actually compute the Dice.
